src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -170,19 +209,30 @@
         {{- form_label(form) -}}
         {{- form_widget(form) -}}
         {{- form_errors(form) -}}
+        {{- block('form_help') -}}
     </{{ element|default('div') }}>
 {%- endblock form_row %}
 
+{# Help #}
+
+{% block form_help -%}
+    {%- for attrname, attrvalue in attr -%}
+        {%- if attrname == 'data-help' -%}
+            <small class="form-text text-muted">{{ attrvalue|raw }}</small>
+        {%- endif -%}
+    {%- endfor -%}
+{%- endblock form_help %}
+
```

Comment:
I agree, that there should be a default way in symfony to add a help message to form elements, but i don't think  `data-help` and this PR is the right place to do this.

Sadly we cannot add a real `help`-Form-Option because that would create conflicts with the dozens of FormExtensions that were created for this purpose?

Reply:
I really like this help block. But I do not think it belongs to this PR. There will be too many discussions about it. 

Please make a new PR with only this change. (Also add the `aria-describedby="emailHelp"` on the input)